### PR TITLE
"Size Limit Exceeded" tip

### PR DESCRIPTION
**Pull request by it33**

### CODE REVIEW

**Comment by coreyhulen** [comment]
+1

File: source/deployment/sso-ldap.md [hunk 1]
```diff
@@ -70,6 +70,6 @@ Note: Currently the value is case sensitive. If the ID attribute is set to the u
 
 This indicates your AD/LDAP server configuration has a maximum page size set and the query coming from Mattermost is returning a result set in excess of that limit. 
 
-To address this issue you can set [max page size](https://docs.mattermost.com/administration/config-settings.html#maximum-page-size) for Mattermost's AD/LDAP configuration to return a sequence of results sets below the max page size set in AD/LDAP, rather than all results in a single query. 
+To address this issue you can set the [max page size](https://docs.mattermost.com/administration/config-settings.html#maximum-page-size) for Mattermost's AD/LDAP configuration. This will return a sequence of result sets below the max page size set in AD/LDAP, rather than returning all results in a single query.
```
Comment (lfbrock) [inline source/deployment/sso-ldap.md]: FYI I changed "results sets" to be "result sets" to be consistent with previous sentence, and broke it into two sentences to make it easier to read.

**Comment by lfbrock** [comment]
+1

**Comment by yangchen1** [comment]
+1

**Comment by esethna** [comment]
Thanks for the reviews, I merged this in a separate commit since there were merge conflicts. https://github.com/mattermost/docs/commit/aba1248e56feb3f542e23f85b4762e150261db79

**Pull request closed without merging**